app/routes/events/view/sessions/list.js

```diff
@@ -75,14 +75,17 @@ export default class extends Route.extend(EmberTableRouteMixin) {
 
     const sessionStatesMapPromise = sessionStateMapCached || this.loader.load('sessions/states');
 
-    const [feedbacks, sessions, sessionStateMap] = await Promise.all([feedbacksPromise, sessionsPromise, sessionStatesMapPromise]);
+    const settingsPromise = this.store.queryRecord('setting', {});
```

Comment:
Why? You need to check your PRs before requesting review if they are ready or contain unneeded changes